src/index.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+// C = K - 273.15
+
+const state = {
+  tempCount: 70,
+  cityName: "",
+  tempValue: null,
+};
+
+//wave 2
+const updateDecreaseTempCount = () => {
+  --state.tempCount;
+  refreshUI()
+  updateTempColorAndLandscape();
+};
+
+const updateIncreaseTempCount = () => {
+  ++state.tempCount;
+  refreshUI();
+  updateTempColorAndLandscape();
+};
+
+const refreshUI = () => {
+  state.tempValue.textContent = `${state.tempCount}°F`;
+};
+
+const updateTempColorAndLandscape = () => {
+  const newLandscape = document.createElement("div");
+  const landscapeContainer = document.getElementById("landscape");
+  landscapeContainer.textContent = "";
+
+  switch (true) {
+    case state.tempCount >= 80:
+      tempValue.style.color = "red";
+      newLandscape.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+      break;
+    case state.tempCount >= 70:
+      tempValue.style.color = "orange";
+      newLandscape.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+      break;
+    case state.tempCount >= 60:
+      tempValue.style.color = "yellow";
+      newLandscape.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+      break;
+    case state.tempCount >= 50:
+      tempValue.style.color = "green";
+      newLandscape.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+      break;
+    case state.tempCount < 50:
+      tempValue.style.color = "teal";
+      newLandscape.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+      break;
+  }
+  landscapeContainer.appendChild(newLandscape);
+};
+
+//Wave 3
+const updateCityNameInput = () => {
+  const cityNameInput = document.getElementById("cityNameInput");
+  const headerCityName = document.getElementById("headerCityName");
+  headerCityName.textContent = cityNameInput.value;
+  state.cityName = cityNameInput.value;
+};
+
+//wave 6
+const resetCityNameInput = () => {
+  cityNameInput.value = "";
+  headerCityName.textContent = "";
+};
+
+//wave 4
+const findTemperature = (coordinates) => {
+  console.log(coordinates)
```

Comment:
Don't forget to remove console logs like this, you could accidentally leak private data. Also just helps to keep the code clutter free.